crates/sample-kmdf-driver/Makefile.toml

```diff
@@ -1 +1,4 @@
 extend = "../../rust-driver-makefile.toml"
+
+[env]
+WDK_BUILD_ADDITIONAL_INFVERIF_FLAGS = "/msft"
```

Comment:
Probably shouldn't list this flag here

Reply:
This is actually required since we use the `Sample` class guid in the `sample-kmdf-driver`, and that guid is reserved only for Microsoft. If we don't have this flag here, we would get the following infverif errors:
```
ERROR(1284) in D:\git-repos\github\windows-drivers-rs.git\worktrees-folder\arm64-improvements\target\debug\sample_kmdf_driver.inf, line 8: Class "Sample" is reserved for use by Microsoft.
ERROR(1285) in D:\git-repos\github\windows-drivers-rs.git\worktrees-folder\arm64-improvements\target\debug\sample_kmdf_driver.inf, line 26: Cannot specify [ClassInstall32] section for Microsoft-defined class.
```

We also [use this flag](https://github.com/microsoft/Windows-driver-samples/blob/64c1afad486ce84bb7d7128afab0aba0fd00f9ef/Build-Sample.ps1#L150) in the [`microsoft/Windows-driver-samples` repo](https://github.com/microsoft/Windows-driver-samples)

Reply:
Also, this is in `Makefile.toml` within the `sample-kmdf-driver` crate, so it only applies to `sample-kmdf-driver`.

This set of commits added functionality for `WDK_BUILD_ADDITIONAL_INFVERIF_FLAGS` which allows downstream users to add additional infverif flags akin to the `InfVerif_AdditionalOptions` property

Reply:
Interesting. I thought it was a private flag since it is undocumented.

Reply:
It is private, and typically shouldn't be used by those outside Microsoft. In this case, we are using it as Microsoft for our driver samples.